web/core/src/jsMain/kotlin/org/jetbrains/compose/web/attributes/AttrsBuilder.kt

```diff
@@ -43,6 +48,14 @@ open class AttrsBuilder<TElement : Element> : EventsListenerBuilder() {
         styleBuilder.apply(builder)
     }
 
+    fun style(propertyHolder: Any) {
```

Comment:
maybe adding one of our @Experimental annotation can be beneficial in this case.

Reply:
UPD: not now, since this PR won't be merged for now